pages/blog/_slug.vue

```diff
@@ -144,6 +154,34 @@ export default {
       ],
     }
   },
+  jsonld() {
+    return {
+      '@context': 'https://schema.org',
+      '@type': 'Article',
+      mainEntityOfPage: {
+        '@type': 'WebPage',
+        '@id': `https://formester.com/blog/${this.article.slug}`,
+      },
+      headline: this.article.title,
+      description: this.article.description,
+      image: this.article.coverImg
+        ? `https://formester.com/blog/cover-images/${this.article.coverImg}`
+        : 'https://formester.com/formester-form-builder-background.png',
+      author: {
+        '@type': 'Person',
+        name: this.article.author,
+      },
+      publisher: {
+        '@type': 'Organization',
+        name: 'Formester',
+        logo: {
+          '@type': 'ImageObject',
+          url: 'https://formester.com/_nuxt/img/logo.5f4f1b9.svg',
```

Comment:
How did you come to this url?

Reply:
okay fine for now I guess but may have to update it as this is not a great way.

Reply:
Updating now